admin/views/tabs/metas/paper-content/post-type-content.php

```diff
@@ -19,6 +19,14 @@
 
 require __DIR__ . '/post_type/post-type.php';
 
+/**
+ * Allow adding custom fields to the admin meta page, just before the archive settings - Post Types tab.
+ *
+ * @api WPSEO_Admin_Pages $yform  The WPSEO_Admin_Pages object
+ * @api string            $name   The post type name
+ */
+do_action( 'Yoast\WP\SEO\admin_post_types_meta_before_archive', $yform, $wpseo_post_type->name );
```

Comment:
Travis fails because this hook name is too long. We should verify if we can shorten it (what about `Yoast\WP\SEO\admin_post_types_archive`?) or add an ignore